microovn/ovn/cmd/commands.go

```diff
@@ -106,11 +107,13 @@ func WaitForDBState(ctx context.Context, _ state.State, db *OvsdbSpec, dbState s
 	return nil
 }
 
-// ovnDBCtl is a helper function to execute "ovn-nbctl" and "ovn-sbctl" commands. It takes "dbType" parameter
-// that identifies which database server it's going to talk to and "args" parameters which is list of
-// arguments that are directly passed to ovn-nbctl/ovn-sbctl commands. Before the command is executed, this
-// function ensures that underlying database is in connected state. If the database does not reach "connected"
-// state before specified timeout, an error is returned.
+// ovnDBCtl is a helper function to execute "ovn-nbctl" and "ovn-sbctl" commands
+// which are tried up to 3 times with a 30s timeout.. It takes "dbType"
```

Comment:
We should mention that the `30s` timeout value is just a fallback in case user didn't specify timeout in CLI args.